public/contents/6-advanced-concepts/tours.json

```diff
@@ -12,11 +12,11 @@
       "fileSize": "23.4 KB",
       "fileName": "introductiontonfiv9.zip",
       "fid": 17807,
-      "checksum": "a618fd697c850ec8c3fe8e4a2923a98186cd582643688f8e00c63e70457755f4",
+      "checksum": "6d8777a2173d1605f31cd8601d1ac2b60a042bb8f90358c54bf694049300b6ba",
       "url": "https://opensource.hcltechsw.com/volt-mx-tutorials/hikes/tour/introduction-nfis-0",
       "alias": "hikes/tour/introduction-nfis-0",
       "description": "Use NFIs to access OS specific functionality, specifically Android Device Settings.",
-      "details": "<p><!--StartFragment--></p> <div> <div> <p>This learning module is a Volt MX HIKE.&nbsp; A HIKE is an interactive tour inside Volt MX Iris that guides you through step-by-step process of each module. You can access the HIKES catalog by clicking the Hike icon in the Iris toolbar.</p> <p>&nbsp;Learn how to:</p> <ul> <li>Configure the Android Native Function APIs required to access Device settings on Android device from an app using Volt MX.</li> </ul> <ul> <li>Add code in the Form controller to access Device Settings</li> </ul> </div> </div> <p><!--EndFragment--></p>",
+      "details": "<p><!--StartFragment--></p><div> <div> <p>This learning module is a Volt MX HIKE.&nbsp; A HIKE is an interactive tour inside Volt MX Iris that guides you through step-by-step process of each module.You can access the HIKES catalog by clicking the Hike icon in the Iris toolbar.</p> <p>&nbsp;Learn how to:</p> <ul> <li>Configure the Android Native Function APIs required to access Device settings on Android device from an app using Volt MX Iris.</li> <li>Add code in the Form controller to access Device Settings</li> </ul> </div> </div> <p><!--EndFragment--></p>",
```

Comment:
@AshokMamidiHcl I think we should have the space after full stop as shown below. Please correct me if I am wrong.

![image](https://user-images.githubusercontent.com/79836554/148173669-791dc6d0-0adf-4c3d-903a-ab0fe0da4230.png)